src/nix/manual.nix

```diff
@@ -0,0 +1,34 @@
+# nix expression to build the Snabb manual.
+#
+# Run like this:
+#   nix-build /path/to/manual.nix
+# ... and the files are produced in ./result/
+
+with import <nixpkgs> {};
+
+stdenv.mkDerivation rec {
+  name = "snabb-manual";
+  src = ./../../.;
+
+  buildInputs = [ ditaa pandoc git
+   (texlive.combine {
+      inherit (texlive) scheme-small luatex luatexbase sectsty titlesec cprotect bigfoot titling droid;
+    })
+  ];
```

Comment:
I'd rather add here:

```
postPatch = ''
  patchShebangs .
'';

This will make sure all shebangs are converted from any form into full path to bash executable. Then we can keep `/usr/bin/env bash` shebangs that should work on all systems including NixOS.